features/twitter.feature

```diff
@@ -10,5 +10,31 @@ Feature: Tweeting Live Events
       | Scrum      | Daily scrum meeting     | Scrum           | 2014/02/03 07:00:00 UTC | 150      | never   | UTC       |         | And an event "Scrum"                      |                       |
     And an event "Scrum"
 
-  Scenario: Event going live causes tweets of hangout link and youtube link to be sent
-    When the HangoutConnection has pinged to indicate the event start, appropriate tweets will be sent
+  Scenario: Event going live without valid live stream does not cause youtube link to be tweeted
+    Given the live stream has not started
```

Comment:
We've used a new way of mocking out the network calls, webmock, to break apart the steps in this scenario, which is a better read.